packages/storage/src/providers/AWSS3Provider.ts

```diff
@@ -446,7 +448,24 @@ export class AWSS3Provider implements StorageProvider {
 				throw error;
 			}
 		}
-
+		if (validateObjectExistence === true) {
```

Comment:
Since it's typed and has even has a default value above, it should be safe to just do a truthy check
```suggestion
		if (validateObjectExistence) {
```

Reply:
Makes sense. Thanks.